README.md

```diff
@@ -235,13 +235,15 @@ Example:
 </plugin>
 ```
 
-## <a name="initScript"></a>Initialization script
-An initialization script file can be provided using the **pathInitScript** parameter of the plugin, in which case it will be executed against the local Elasticsearch cluster.
+## <a name="initScripts"></a>Initialization scripts
+A comma-separated list of initialization script files can be provided using the **pathInitScript** parameter of the plugin, in which case they will be executed against the local Elasticsearch cluster.
 The file extension defines the file format: *json* for JSON format, anything else for custom format.
 
+The script paths will be trimmed and executed in the order they are provided.
+
```

Comment:
Actually, can you add an example here? It would be clearer how this works.

Reply:
Done